### PR TITLE
fix: Standardize the height of the universal nav

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

ref: https://github.com/freeCodeCamp/freeCodeCamp/issues/52119

Screenshot 2023-10-26 at 3 04 50 PM

### CODE REVIEW

File: src/_includes/assets/css/screen.css [hunk 1]
```diff
@@ -172,13 +172,13 @@ body {
   z-index: 300;
   display: flex;
   justify-content: space-between;
-  align-items: flex-start;
+  align-items: center;
   /* overflow-y: auto; */
   height: 40px;
```
Comment (ahmaxed) [inline src/_includes/assets/css/screen.css]:
```suggestion
```

File: src/_includes/assets/css/screen.css [hunk 2]
```diff
@@ -61,7 +61,7 @@ body {
 .site-main {
   z-index: 100;
   flex-grow: 1;
-  margin-top: calc(var(--header-height) * 3 + 5px);
+  margin-top: calc(var(--header-height) * 2 + 20px);
```
Comment (ShaunSHamilton) [inline src/_includes/assets/css/screen.css:64]: Why change this?

**Review by ShaunSHamilton — COMMENTED** [review]
I am not sure the best way to review this (plugged the CSS into the dev-tool declarations). Seems good to me.

_Also wanted to say I spent a good few minutes wondering why this PR was not working on the forum_ 🤦‍♂️

**Comment by ahmaxed** [comment]
I think @scissorsneedfoodtoo would be the best person to review this.

File: src/_includes/assets/css/screen.css (hunk 2 above)
Comment (scissorsneedfoodtoo) [reply src/_includes/assets/css/screen.css:64]: My understanding is that this adds some spacing between the nav + donation banner and the rest of the page. Now that the header height has been increased, this formula needed to be adjusted so the spacing remains about the same.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Everything LGTM.

I did a build of Japanese News and compared the height of the navbar to the one on Japanese Learn. With this updated styling, the height of the navbar matches the one on Learn on mobile, tablet, and desktop sized screens.

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
Sorry, spoke a bit too soon.

Looking at the author and tag pages, it seems like there's a bit of a gap between the donor banner and fCC blue background:

image

In `screen.css` on line 122, changing `margin-top` from `calc(var(--header-height) * 2)` to `calc(var(--header-height) * 1.7)` seems to do the trick.

Would you be able to add that to this PR, @ahmaxed?

File: src/_includes/assets/css/screen.css (hunk 2 above)
Comment (ahmaxed) [inline src/_includes/assets/css/screen.css:64]:
```suggestion
  margin-top: calc(var(--header-height) * 1.7);
```

**Comment by ahmaxed** [comment]
> In `screen.css` on line 122, changing `margin-top` from `calc(var(--header-height) * 2)` to `calc(var(--header-height) * 1.7)` seems to do the trick.
> 
> Would you be able to add that to this PR, @ahmaxed?

Thanks for taking a look. I added your suggestion to the margin-top.

File: src/_includes/assets/css/screen.css [hunk 3]
```diff
@@ -61,7 +61,7 @@ body {
 .site-main {
   z-index: 100;
   flex-grow: 1;
-  margin-top: calc(var(--header-height) * 3 + 5px);
+  margin-top: calc(var(--header-height) * 1.7);
```
Comment (scissorsneedfoodtoo) [inline src/_includes/assets/css/screen.css]: Going back to the original suggestion:

```suggestion
  margin-top: calc(var(--header-height) * 2 + 20px);
```

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Hey @ahmaxed, just checked this out and unfortunately the latest change doesn't give enough spacing between the navbar + donation banner in all views, and the date in posts just barely clears the donation banner now.

Sorry, I may not have been clear enough in my last comment.

I'd recommend going back to `margin-top: calc(var(--header-height) * 2 + 20px)` in the `.site-main` ruleset. That looked fine the way it was.

And I'm not able to select it and leave a suggestion, but on [line 122](https://github.com/freeCodeCamp/news/blob/e0eef89e0dc6e3cb301fedcab2d0715d5c599bd5/src/_includes/assets/css/screen.css#L122) within the `.under-header-content` ruleset, if you can change that to `margin-top: calc(var(--header-height) * 1.7)`, that should fix the spacing issue on tag and author pages.

**Comment by ahmaxed** [comment]
Sorry for the confusion. This should do it.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Thanks for your patience, @ahmaxed, and for making that update.

Just checked locally and everything LGTM 👍

**Pull request merged**